_posts/2017-04-24-get-electrified-3.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+comments: true
+
+title: Get Electrified 3
+excerpt: Tinkering with Digital Electronics!
+author: Mayanka Medhe
+category: [Informative]
+tags: [Digital Electronics, Adder, MUX]
+
+assets-dir: assets/get-electrified-3
+header-img: "assets/get-electrified-3/cover.jpg"
+---
+
+Electronics club conducted its last electrified session on 18th March, 2017. It was totally awesome as we got an opportunity to make a half adder, full adder, subtractor and priority encoder. Along with that we also learnt some cool stuff about digital circuits. 
+
+ 
+
+**Where do we begin from to make anything as simple as an adder?**
+
+We basically use a combinational circuit. Not only an adder but we can even make a subtractor, priority encoder, MUX and what not!
+
+Combinational circuit is basically used for making those circuits which depends on the current state.
+
+![Combinational Logic Circuits]({{ site.baseurl }}/{{ page.assets-dir }}/image_0.png)
+
+**So, let us now begin with how exactly do these combinational circuits function:**
+
+** Adder:**
```

Comment:
This line doesn't get rendered as bold, maybe change it to `**Adder:**`. Or just make it a subheading (`### Adder`).